configs/fsdp/baseline.yaml

```diff
@@ -0,0 +1,55 @@
+MODEL:
```

Comment:
Please rename the file to example.yaml

Reply:
I think we start with just example.yaml for now.